packages/expo-av/android/src/main/java/expo/modules/av/progress/ProgressLooper.kt

```diff
@@ -0,0 +1,74 @@
+package expo.modules.av.progress
+
+typealias TimeMachineTick = () -> Unit
+
+interface TimeMachine {
```

Comment:
I don't like this name TBH 😛
What `TimeMachine` means?
I dunno what might be better, but maybe sth that connects with JS's `Timer`/`Timeout` or Java's/Kotlin's `Handler` or `Timer` from [this SO answer](https://stackoverflow.com/a/43348715/4337317)

Reply:
It advences you in time by a given period. What better name for that?